src/containers/AffectiveRatingContainer.tsx

```diff
@@ -1,43 +1,86 @@
-import { ExperimentModule } from './ExperimentContainer'
+import { ExperimentModule, VisualStimuli } from './ExperimentContainer'
 import { AffectiveRatingScreen } from '@screens'
-import { ImageSourcePropType } from 'react-native'
+import { shuffle } from 'lodash'
```

Comment:
Suggestion: Do `import shuffle from 'lodash/shuffle'`

This isn't super important because this is react-native, but importing it that way makes it so that only the necessary code is included in the build, and not the entirety of lodash.

This is more of a tip really for working with lodash on web projects.